src/controllers/krate/metadata.rs

```diff
@@ -243,13 +264,15 @@ impl FromStr for ShowIncludeMode {
                         categories: true,
                         badges: true,
                         downloads: true,
+                        default_version: true,
```

Comment:
I guess it's a bit unfortunate that we named the legacy default value `full`. I think I'd prefer if we did not add more fields to the default case where `?include=...` is not set 😅

Reply:
If we share the same key, I think we could simply make `default_version` default to `true`, right? Since it's effectively shadowed by `versions` by default?

Reply:
yep, and the next comment would also become obsolete since there is no new key that would be added